zum-front-end/src/view/components/card.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Title: Card Component
+ * Description: Card Component for subpage
+ * Author : Seokhyeon Jang (coolman555@me.com)
+ */
+
+import zumStore from "../../store/index";
+import "./card.css";
+
+export default {
+  el: "#card",
+  data() {
+    return {
+      update: false
+    }
+  },
+  methods: {
+    // Add bookmark in the store
+    bookmark(data) {
+      zumStore.commit.bookmarker((<any>this).component.list.find(element => element.idx === data));
+      (<any>this).setState({ update: !(<any>this).state.update });
```

Comment:
`<any>this`를 `this`로 사용할 수 있도록 코어를 구성해보세요!
그리고 이러한 부분은
```suggestion
      const $this = <any>this;
      zumStore.commit.bookmarker($this.component.list.find(element => element.idx === data));
      $this.setState({ update: !$this.state.update });
```
이런식으로 사용하면 더 깔끔하게 작성할 수 있습니다.